### PR TITLE
activity composer link in dropdown admin menu

### PR DESCRIPTION
**Pull request by slickexo2**

### CODE REVIEW

File: war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<node-navigation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                 xsi:schemaLocation="http://www.gatein.org/xml/ns/gatein_objects_1_3 http://www.gatein.org/xml/ns/gatein_objects_1_3"
+                 xmlns="http://www.gatein.org/xml/ns/gatein_objects_1_3">
+    <priority>1</priority>
+    <page-nodes>
+        <node>
+            <uri>functional-composer</uri>
```
Comment (rdenarie) [inline war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml:8]:
```suggestion
            <uri>functional-configuration</uri>
```

File: war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+            <name>functional-composer</name>
```
Comment (rdenarie) [inline war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml:9]:
```suggestion
            <name>functional-configuration</name>
```

File: war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            <label>Functional Composer</label>
```
Comment (rdenarie) [inline war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml:10]:
```suggestion
            <label>Functional Configuration</label>
```

File: war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+            <label xml:lang="en">Functional Composer</label>
```
Comment (rdenarie) [inline war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml:11]:
```suggestion
            <label xml:lang="en">Functional Configuration</label>
```

File: war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+            <label xml:lang="fr">Functional Composer</label>
```
Comment (rdenarie) [inline war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/portal/setup-navigation.xml:12]:
```suggestion
            <label xml:lang="fr">Functional Configuration</label>
```

File: war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/menu-configuration.xml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+    <external-component-plugins>
+        <target-component>org.exoplatform.platform.common.service.MenuConfiguratorService</target-component>
+        <component-plugin>
+            <name>Activity Configuration Plugin</name>
+            <set-method>addNavigation</set-method>
+            <type>org.exoplatform.platform.common.service.plugin.MenuConfiguratorAddNodePlugin</type>
+            <init-params>
+                <value-param>
+                    <name>extended.setup.navigation.file</name>
+                    <value>war:/conf/activity-composer-configuration/portal/setup-navigation.xml</value>
```
Comment (rdenarie) [inline war/src/main/webapp/WEB-INF/conf/activity-composer-configuration/menu-configuration.xml:16]:
```suggestion
                    <value>war:/conf/functional-configuration/portal/setup-navigation.xml</value>
```
We should rename the file with the new name of the addon.

**Review by rdenarie — CHANGES_REQUESTED** [review]
Hello,
Please can you update labels and configuration file name to be homogeneous ?

Thanks !

**Comment by rdenarie** [comment]
@jpbriand I think we can close this PR as it is merged in the global one.
Can you confirm ?
Thx

**Comment by slickexo2** [comment]
@rdenarie  You can close this PR.
All the corrections are present in the other one.

**Pull request closed without merging**